integration-tests/quartz/src/main/java/io/quarkus/it/quartz/DisabledScheduledMethods.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.it.quartz;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import io.quarkus.scheduler.Scheduled;
+
+@ApplicationScoped
+public class DisabledScheduledMethods {
+    static String valueSetByCronScheduledMethod = "";
```

Comment:
I think that we should make these variables `volatile`...

Reply:
Same reasoning as https://github.com/quarkusio/quarkus/pull/25408#discussion_r870279981

Reply:
I see your point. I've added the suggestion and added some comments to the methods